test/test_misc_utils.py

```diff
@@ -3159,6 +3159,16 @@ def test_dict_to_keys_and_values():
     assert dict_to_keys_and_values({}) == []
 
 
+def test_is_c4_arn():
+    assert is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-cgap-production-stack-Productionblue-fzOm82DGZKGZ')
+    assert is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-cgap-production-stack-CgapProductionblue-fzOm82DGZKGZ')
+    assert is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-fourfront-hotseat-stack-Hotseat-ZnIAAAz8Apxf')
+    assert is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-fourfront-production-stack-FourfrontProd-hU0JiwOeWfdm')
+    assert not is_c4_arn('arn:aws:ecs:us-east-1:643:cluster/c4-ecs-abc-webdev-stack-CgapWebdev-rSLwZBbdVTtx')
```

Comment:
Perhaps we should use a regexp match and look for "[:/]c4-" in the name as well?

Reply:
You mean in the `is_c4_arn` function?

Reply:
(and doing re.search, not re.match)

Reply:
Got it, thanks.